UnityProject/Assets/Scripts/UI/IngameMenu/SetVolume.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+public class SetVolume : MonoBehaviour
+{
+		public AudioMixer mixer;
```

Comment:
Not everything is using the mixer, that is why I suggested changing the AudioListeners.Volume property.